### ISSUE TITLE
XML ValueError: can only parse strings

### ISSUE THREAD

**Issue by furlongm** [furlongm/patchman#30]
/etc/cron.daily/python-patchman:
patchman.localdomain     : gunzip: Error -3 while decompressing data: incorrect header check
Traceback (most recent call last):
  File "/usr/bin/patchman", line 593, in <module>
    main()
  File "/usr/bin/patchman", line 588, in main
    showhelp = process_args(args)
  File "/usr/bin/patchman", line 578, in process_args
    update_errata(args.force)
  File "/usr/lib/python2.7/dist-packages/patchman/packages/utils.py", line 85, in update_errata
    parse_errata(data, force)
  File "/usr/lib/python2.7/dist-packages/patchman/packages/utils.py", line 100, in parse_errata
    result = etree.XML(data)
  File "src/lxml/lxml.etree.pyx", line 3192, in lxml.etree.XML (src/lxml/lxml.etree.c:82678)
  File "src/lxml/parser.pxi", line 1818, in lxml.etree._parseMemoryDocument (src/lxml/lxml.etree.c:124513)
ValueError: can only parse strings
run-parts: /etc/cron.daily/python-patchman exited with return code

**Comment by maggon** [comment]
@furlongm do you have a resolution for this issue? I am seeing this error too.

**Comment by eberkut** [comment]
I pinpointed this issue to be related to the -a argument including --errata. The documentation is not up-to-date on this.

I've removed --errata from -a by commenting this line https://github.com/furlongm/patchman/blob/41dc78606398f23c8cd66e265c0f254a79581265/sbin/patchman#L541 and the issue disappeared.

I don't need to get CentOS errata so I haven't properly analyzed the root cause in utils.py which causes lxml to fail to uncompress the errata file.
